profile/2019-03-01-hybrid.json

```diff
@@ -6,175 +6,472 @@
     "resource-manager": {
         "microsoft.authorization": {
             "2016-09-01": [
-                "locks"
+                {
+                    "resourceName": "locks",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2016-09-01/locks.json"
+                }
             ],
             "2016-12-01": [
-                "policyDefinitions",
-                "policyAssignments"
+                {
+                    "resourceName": "policyDefinitions",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2016-12-01/policyDefinitions.json"
+                },
+                {
+                    "resourceName": "policyAssignments",
+                    "path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/resources/resource-manager/Microsoft.Authorization/stable/2016-12-01/policyAssignments.json"
+                }
             ],
             "2015-07-01": [
-                "operations",
-                "permissions",
-                "roleAssignments",
-                "roleDefinitions",
-                "providerOperations"
+                {
+                    "resourceName": "permissions",
```

Comment:
Gov Cloud, wanted to add another property here, AvailableClouds to indicate what are all the clouds that supports this resource type @sijuman

Reply:
@bganapa yes that is better. If you think any new meta data makes the current spec clearer to interpret the intention that is great news.
Feel free to add more meta data that will help to make this spec clearer.